src/backend/crossterm.rs

```diff
@@ -86,7 +86,7 @@ where
         for (x, y, cell) in content {
             // Move the cursor if the previous location was not (x - 1, y)
             if !matches!(last_pos, Some(p) if x == p.0 + 1 && y == p.1) {
-                map_error(queue!(self.buffer, MoveTo(x, y)))?;
+                queue!(self.buffer, MoveTo(x, y))?;
```

Comment:
I see this is removed a bunch of times. Can you please find out or explain why this was necessary in the first place?

Reply:
It seems like the `queue` macro, or `QueueableCommand::queue`, now return `io::Result` making the `map_error` unnecessary.